docs/backend/setup/_category_.json

```diff
@@ -0,0 +1,6 @@
+{
+  "label": "Setup",
+  "position": 4,
+  "collapsed": true
+}
+
```

Comment:
This is a placeholder for when this gets merged in so we can begin with other sections. Backend Setup cannot easily be worked on until this PR is merged.